packages/job-manager/src/worker/worker.ts

```diff
@@ -38,8 +38,8 @@ export class Worker {
                 const scanMessages = await this.getMessages(tasksIncrementCount);
                 if (scanMessages.length === 0) {
                     this.logger.logInfo(`The storage queue '${this.queue.scanQueue}' has no message to process.`);
-                    if (poolMetricsInfo.load.activeTasks === 0) {
-                        this.logger.logInfo(`Exiting the ${this.jobId} job since there are no active tasks.`);
+                    if (poolMetricsInfo.load.activeTasks === 0 && poolMetricsInfo.load.runningTasks === 0) {
```

Comment:
uts missing

Reply:
(u should have a test case where active tasks is > 0 & running tasks =  0 & vise versa